rfps/ethereum-test-vectors.md

```diff
@@ -0,0 +1,101 @@
+2. Deploy these contracts to the Filecoin testnet (called Butteflynet).
+3. Fetch thousands of transactions sent to these contracts on Ethereum in the past. 
+4. Replay these transactions to contracts deployed on Filecoin. 
+5. Compare the Ethereum state before and after these transactions with the results on Filecoin. We expect the team to research the ideas of how to do state comparison reliably.
+
+We want to run this tool a few times if needed and fix encountered errors every time.
+This tool can be written in either Go or Rust.
+
+## List of deliverables
+
+We expect the team to provide a Github repository containing
+* Source code of the tool
+* Guides on how to use it
+* Fetched smart contracts and transactions so the same data doesn’t have to be re fetched multiple times
+
+We also expect a demo of running all steps against the Wallaby (TBD) testnet to prove it works.
+
+## Timeline
+
+We expect this work to be done till 7th of November. We want to be able to use this tool after we deploy FVM M2.1 to Butterflynet. This is scheduled for 14th of November 2022.
+
+## Milestones & Funding
+
+<table>
+  <tr>
+   <td><strong>Milestone No.</strong>
+   </td>
+   <td><strong>Milestone Description</strong>
+   </td>
+   <td><strong>Funding</strong>
+   </td>
+   <td><strong>Estimated Timeframe</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>1
+   </td>
+   <td>Implementation of a tool which fetches hundreds of contracts from Ethereum, stores them and and deploys them to Filecoin testnet
+   </td>
+   <td>TBD
+   </td>
+   <td>5 days
+   </td>
+  </tr>
+  <tr>
+   <td>2
+   </td>
+   <td>Implementation of a tool which fetches thousands of transactions from Ethereum, stores them and replays them on Filecoin testnet
+   </td>
+   <td>TBD
+   </td>
+   <td>5 days
+   </td>
+  </tr>
+  <tr>
+   <td>3
+   </td>
+   <td>Implementation of a tool which compares the Ethereum state before and after these transactions with the results on Filecoin
+   </td>
+   <td>TBD
+   </td>
+   <td>8 days
+   </td>
+  </tr>
+  <tr>
+   <td>4
+   </td>
+   <td>How-to guides are prepared and demo is presented
+   </td>
+   <td>TBD
+   </td>
+   <td>2 days
+   </td>
+  </tr>
+</table>
+
+## Recommended Team
+
+Please provide links to your team members' Githubs or LinkedIn profiles.
+
+* A team with strong design and web development skills (please show evidence of this in your proposal) and experience creating production blockchain applications
+* Experience extracting metadata from a blockchain or a complex tech back-end
+* Solid knowledge of Filecoin chain data structures
+
+## How to apply
+
+To submit a grant proposal, please open a <a href="https://github.com/filecoin-project/devgrants/issues/new?assignees=realChainLife&labels=RFP&template=rfp-application.md&title=RFP+Application">new RFP issue</a> in the `filecoin-project/devgrants` repo.
+For questions or to submit a private proposal you can also email eva [at] protocol.ai.
```

Comment:
@maciejwitowski - are you the technical sponsor for this? If so can we add your contact info here instead?

Reply:
Sure, good point!